### PR TITLE
fix broken link pointing to rust-lang-nursery -> rustwasm

### PR DESCRIPTION
**Pull request by epsilonhalbe**

### CODE REVIEW

File: _posts/2018-04-05-welcome-to-this-week-in-rust.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This is a weekly summary of its progress and community.
 
 Tweet us at [@rustwasm](https://twitter.com/rustwasm) or send us a pull request. Want to get involved? [We love contributions!][get-involved]
 
-[get-involved]: https://github.com/rust-lang-nursery/rust-wasm/blob/master/README.md#get-involved
+[get-involved]: https://github.com/rustwasm/rust-wasm/blob/master/README.md#get-involved
```
Comment (sendilkumarn) [inline _posts/2018-04-05-welcome-to-this-week-in-rust.md]: can you change it to `rustwasm/team` ?

File: _posts/2018-04-21-this-week-in-rust-wasm-001.markdown [hunk 2]
```diff
@@ -11,16 +11,16 @@ Hello and welcome to another issue of *This Week in Rust and WebAssembly*!
...
 
 ## News
 * [WebAssembly Studio - Online IDE](https://hacks.mozilla.org/2018/04/sneak-peek-at-webassembly-studio/)
-* New, in-depth tutorial: [Conway's Game of Life in Rust and WebAssembly](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/introduction.html). Topics covered include:
-    * [Setting up the toolchain and making a hello world](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/setup.html)
-    * [Designing Rust programs for WebAssembly](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/implementing.html)
-    * [Debugging](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/debugging.html)
-    * [Time profiling](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/time-profiling.html)
-    * [Code size profiling](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/code-size.html)
+* New, in-depth tutorial: [Conway's Game of Life in Rust and WebAssembly](https://rustwasm.github.io/rust-wasm/game-of-life/introduction.html). Topics covered include:
```
Comment (sendilkumarn) [inline _posts/2018-04-21-this-week-in-rust-wasm-001.markdown]: these links are wrong `/book` I think

File: _posts/2018-04-05-welcome-to-this-week-in-rust.md (hunk 1 above)
Comment (sendilkumarn) [reply _posts/2018-04-05-welcome-to-this-week-in-rust.md]: change it everywhere

**Comment by epsilonhalbe** [comment]
I think I have fixed the book links and the team link.

**Pull request merged**